cmd/gb-vendor/fetch.go

```diff
@@ -56,7 +56,11 @@ Flags:
 			return err
 		}
 
-		wc, err := repo.Checkout(branch, revision)
+		if m.HasDependencyWithUrl(repo.URL()) {
+			return fmt.Errorf("specified repository is already downloaded")
```

Comment:
If you open the pull request, there is "Show outdated diff" next to the "davecheney commented on an outdated diff an hour ago".